util-images/network/netperfbenchmark/go.mod

```diff
@@ -0,0 +1,7 @@
+module k8s.io/perf-tests/util-images/network/netperfbenchmark
+
+go 1.12
```

Comment:
Can we go higher than 1.12 - it's already out of support window IIRC

Can we use 1.15?

Reply:
Changed it to 1.15.